tests/general/LineContinuationBearTest.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from queue import Queue
+
+from bears.general.LineContinuationBear import LineContinuationBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+good_file = """
+a = some_function(
+    '1' + '2')
+"""
+
+bad_file = r"""
+a = '1' \
```

Comment:
this bear is about banning use of line continuation in our codebase.  that includes test modules.  Please remove the use of line continuation here.

This was previously mentioned at https://coala.zulipchat.com/#narrow/stream/101940-coala-bears/subject/line.20continuation.20bear/near/147283234

It can be used in external sample files which are not our codebase.

Reply:
Oh, okay. I understand it now @jayvdb :+1: